airflow/utils/log/arion_log_formatter.py

```diff
@@ -0,0 +1,34 @@
+import logging
+
+class ArionFormatter(logging.Formatter):
+
+    err_fmt  = "ERROR: %(msg)s"
+    dbg_fmt  = "DBG: %(module)s: %(lineno)d: %(msg)s"
+    info_fmt = "%(msg)s"
+
+    def __init__(self):
+        super().__init__(fmt="%(levelno)d: %(msg)s", datefmt=None, style='%')
+
+    def format(self, record):
+
+        # Save the original format configured by the user
+        # when the logger formatter was instantiated
+        format_orig = self._style._fmt
+
+        # Replace the original format with one customized by logging level
+        if record.levelno == logging.DEBUG:
+            self._style._fmt = ArionFormatter.dbg_fmt
```

Comment:
should ArionFormatter.dbg_fmt be self.dbg_fmt? same question for below

Reply:
these are Class variables and not inside the constructor, I am sure this will throw an error if replace it with self. Any thoughts @dineshanchan and @chaithra-yenikapati?

Reply:
Okay I tested with `self. dbg_fmt` works fine. Thanks for pointing this out @moyphilip learned something new 👍